weblate/templates/mail/account_activity.html

```diff
@@ -29,9 +29,9 @@
 </table>
 
 <p>
-{% blocktrans %}If you don’t recognize this change, you can report it by clicking the button below.{% endblocktrans %}
-<b>{% blocktrans %}Don’t use this button for support questions. Such questions won’t be answered.{% endblocktrans %}</b>
-{% blocktrans with link_start='<a href="https://github.com/WeblateOrg/weblate/issues/new/choose">' link_end='</a>'  %}If you have an idea or a question, {{ link_start }}this{{ link_end }} is a good place to start.{% endblocktrans %}
+{% blocktrans %}If this was not done by you, report it by clicking the button below.{% endblocktrans %}
```

Comment:
Succinct avenue to grasp the problem, or in the very least its solution, and then be done.